pkg/minikube/cluster/cluster.go

```diff
@@ -104,8 +104,8 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 
 	if h.Driver.DriverName() != config.VMDriver {
 		out.T(out.Empty, "\n")
-		out.WarningT(`Ignoring --vm-driver={{.driver_name}}, as the existing "{{.profile_name}}" VM was created using the {{.driver_name2}} driver.`,
-			out.V{"driver_name": config.VMDriver, "profile_name": cfg.GetMachineName(), "driver_name2": h.Driver.DriverName()})
+		out.WarningT(`The existing "{{.profile_name}}" VM was created using the {{.driver_name}} driver.`,
```

Comment:
I like the way you worded this.

The description in #4998 was not very clear, but to mirror my intent, could you change this PR to:

- Omit all of these warnings if the driver flag was not set by the user: `if !cmd.Flags().Changed(vmDriver)`
- If the driver flag was set, call `exit.WithCode(exit.Config, "<this message>")`

Reply:
sure

Reply:
@tstromberg PTAL.